install-scripts/install-macos.sh

```diff
@@ -26,4 +26,3 @@ source ~/cairo_venv/bin/activate
 
 # Install cairo & its dependencies
 pip3 install -r requirements.txt
-
```

Comment:
I think we should install `pyenv install 3.9.15`

Reply:
Also, this isn't duplicated? Since then `install_macos` runs `make deps-macos`
But we can leave it duplicate because `make deps-macos` won't fail.

What we need to check is to add the `CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib` before the pip install

Reply:
Good catch! Removed the whole Python install 👍